overlord/configstate/configmgr.go

```diff
@@ -55,6 +55,11 @@ func Init(st *state.State, hookManager *hookstate.HookManager) error {
 	}
 	dirs.SetSnapHomeDirs(homedirs)
 
+	// cleanup old experimental features
+	if err := configcore.CleanExperimentalFlags(tr); err != nil {
```

Comment:
Does this happen before or after we compute the system key?

Reply:
The cleanup happens before computing the system key, specifically when calling:
```
cmd/snapd
  main()
    daemon.New()
      overlord.New()
         configstateInit()
```
where the system-key is comuted during Start calls:
```
cmd/snapd
  main()
    d.Start()
        d.overlord.StartUp()
          o.stateEng.StartUp()
            ifacestate.StartUp()
              regenerateAllSecurityProfiles()
```